modules/uhsg_chat/uhsg_chat.module

```diff
@@ -27,4 +27,9 @@ function uhsg_chat_page_attachments(array &$attachments) {
       $attachments['#attached']['drupalSettings']['uhsg_chat']['currentLanguage'] = \Drupal::languageManager()->getCurrentLanguage()->getId();
     }
   }
+
+  // The Giosg chat script is added to every page, but activated in specific
+  // paths remotely in giosg console. Currently no settings defined in Drupal.
+  $attachments['#attached']['library'][] = 'uhsg_chat/giosg_chat';
```

Comment:
It might be better to add the attachment only when viewing a node, so not to load it when we do not need it (such as on admin interface etc.).

Reply:
Changed.